packages/dataviews/src/components/dataviews-item-actions/index.tsx

```diff
@@ -106,18 +106,20 @@ export function ActionModal< Item >( {
 	const label =
 		typeof action.label === 'string' ? action.label : action.label( items );
 	return (
-		<Modal
-			title={ action.modalHeader || label }
-			__experimentalHideHeader={ !! action.hideModalHeader }
-			onRequestClose={ closeModal }
-			focusOnMount={ action.modalFocusOnMount ?? true }
-			size={ action.modalSize || 'medium' }
-			overlayClassName={ `dataviews-action-modal dataviews-action-modal__${ kebabCase(
-				action.id
-			) }` }
-		>
-			<action.RenderModal items={ items } closeModal={ closeModal } />
-		</Modal>
+		<Suspense fallback={ null }>
```

Comment:
I considered allowing a fallback element to be passed in.

We have other props on the `ActionModal` interface for customising the modal e.g. `modalHeader`, `modalSize`, etc. So a `modalSuspenseFallback` wouldn't necessarily be out of place.